<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddComputerWindow</class>
 <widget class="QDialog" name="AddComputerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Add Computer</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="nameInput"/>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QRadioButton" name="wasBuiltInput">
     <property name="text">
      <string>Was it built?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="yearBuiltLabel">
     <property name="text">
      <string>Year Built:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="yearBuiltInput">
     <property name="placeholderText">
      <string>Leave empty if the computer was never built</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="typeLabel">
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="typeInput">
     <item>
      <property name="text">
       <string>Transistor</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Electromechanical</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mechanical</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Electronic</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="addButton">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
